specification.md

```diff
@@ -0,0 +1,1199 @@
+  </tr>
+  <tr>
+    <td>range</td>
+    <td>Optional</td>
+    <td>When the responses are to be visualized on a scale, provides the minimum and maximum relevant values of the range.</td>
+    <td>{'range':[0,10]}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### open
+
+Represents a Response that might be in one of several formats. This Question type is useful for representing open-ended Responses to Flows that run over multiple channels (IVR, SMS, social media) and allow the Contact to submit an audio message, image, video, or text response.
+
+#### Response format
+
+Response must be in the format required for the Question type of each row (where `type` is identified within the response metadata.)
+
+#### Type options (type_options)
+
+None used at the`schema` level.  (Refer to the `type_options` within each row.)
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`type`</td>
+    <td>Yes</td>
+    <td>Must be one of the other supported question types (e.g., `text`, `audio`, `image`, etc.)</td>
+    <td>'text'
+
+'image'</td>
+  </tr>
+  <tr>
+    <td>`type_options`</td>
+    <td>Yes</td>
+    <td>Includes the schema metadata (normally found in the schema) that would be used for that response row.</td>
+    <td>{}
+
+{'format':'png'}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td>For additional response metadata, refer to the details for each respective question type.</td>
+    <td></td>
+  </tr>
+</table>
+
+
+### text
+
+Represents any arbitrary text response.
+
+#### Response format
+
+A string:
+
+```
+"I am learning this specification."
+```
+
+#### Type options (type_options)
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+#### Response Metadata
+
+<table>
+  <tr>
+    <td>Object</td>
+    <td>Required</td>
+    <td>Details</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`language`</td>
+    <td>Optional</td>
+    <td>The ISO 639-3 code for the language of the response, if known.</td>
+    <td>{'language':'eng'}</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+### image
+
+Represents a picture submitted by the Contact.
+
+#### Response format
+
+A string with the URL where the image can be retrieved. (TODO: Do we want to support inline image data?)
```

Comment:
Let's discuss this on the call tomorrow - I just made Issue #2: "Consider inline embedded media data in the JSON data file"